src/xenia/app/emulator_window.cc

```diff
@@ -174,10 +182,71 @@ bool EmulatorWindow::Initialize() {
   // FIXME: This code is really messy.
   auto main_menu = MenuItem::Create(MenuItem::Type::kNormal);
   auto file_menu = MenuItem::Create(MenuItem::Type::kPopup, L"&File");
+  // Recent Sub tap
+  auto recent_list = MenuItem::Create(MenuItem::Type::kPopup, L"&Open Recent");
+  //******Add Recent PATHS to menu***************************************
+  std::string LINES[10];
+  std::ifstream RECENT_FILE("recent.txt");
```

Comment:
There's things to be said about how you're creating this file here, but I will leave this out until other problems are resolved.

Reply:
ifstream is used to read the file if it exists. If it doesn't then ofstream creates it when a file is opened. Is there still a better way of doing this, if so how?